README.md

```diff
@@ -35,6 +35,22 @@ EventMachine.run do
 end
 ```
 
+To pass additional parameters while subscribing, add them to the identifier hash. These will be available server side in the channel's params hash.
```

Comment:
it can, here are some examples: https://gist.github.com/NullVoxPopuli/edfcbbe91a7877e445cbde84c7f05b37